modules/data-loading/pages/kafka-loader-user-guide.adoc

```diff
@@ -31,75 +118,118 @@ There are three basic steps:
-
 After the data source is created, then use the `SET` command to specify the path to a configuration file for that data source.
+Each time when the config file is updated, you must run `SET example_data_source` to update the data source details in the dictionary.
 
 [source,gsql]
 ----
-GSQL > SET data_source_name = "/path/to/kafka.config"
+GSQL > SET example_data_source = "/path/to/kafka.config" <1>
 ----
+<1> If you have a TigerGraph cluster, the configuration file must be on machine m1, where the GSQL server and GSQL client both reside,  and it must be in JSON format. If the configuration file uses a relative path, the path should be relative to the GSQL client working directory.
 
-This SET command reads, validates, and applies the configuration file, integrating its settings into TigerGraph's dictionary. The data source configuration file's content, structured as a JSON object, describes the Kafka server's global settings, including the data source IP and port. A sample kafka.conf is shown in the following example:
+For simplicity, you can merge the `CREATE DATA_SOURCE` and `SET` statements:
 
-[source,text]
+[source.wrap,gsql]
 ----
-{
-  "broker": "192.168.1.11:9092",
-}
+GSQL > CREATE DATA_SOURCE KAFKA data_source_name = "/path/to/kafka.config"
 ----
 
-The "broker" key is required, and its value is composite of a fully qualified domain name (or IP) and port. Additional Kafka configuration parameters may be provided (see Kafka documentation) by using the optional `"kafka_config"` key. For its value, provide a list of key-value pairs. For example:
+To further simplify, instead of specifying the Kafka data source config file path, you can also directly provide the Kafka data source configuration as a string argument, as shown below:
 
-[source,text]
+[source,gsql]
 ----
-{
-  "broker": "localhost:9092",
-  "kafka_config": {"group.id":"tigergraph"}
-}
+GSQL > CREATE DATA_SOURCE KAFKA data_source_name = "{\"broker\":\"broker.full.domain.name:9092\"}"
 ----
 
-For simplicity, you can merge the `CREATE DATA_SOURCE` and `SET` statements:
+[TIP]
+====
+The above simplified statement is useful for using Kafka Data Loader in TigerGraph Cloud. In TigerGraph Cloud (tgcloud.io), you can use GSQL web shell to define and create Kafka data sources, without creating the Kafka data source configuration file in filesystem.
+====
+
+
+[#_2_create_a_loading_job]
+=== 2. Create a Loading Job
+
+The Kafka Loader uses the same basic https://docs.tigergraph.com/dev/gsql-ref/ddl-and-loading/creating-a-loading-job#create-loading-job[CREATE LOADING JOB] syntax used for standard GSQL loading jobs.A `DEFINE FILENAME` statement should be used to assign a loader `FILENAME` variable to a Kafka data source name and the path to its config file.
+
+In addition, the filename can be specified in the `RUN LOADING JOB` statement with the `USING` clause.The filename value set by a `RUN` statement overrides the value set in the `CREATE LOADING JOB`.
```

Comment:
Missing space at the end of the sentence: "...GSQL loading jobs.A `DEFINE FILENAME` statement..."
The same here: "...with the `USING` clause.The filename value set..."

Should each sentence be on a separate Antora line anyway?